curation_portal/auth.py

```diff
@@ -1,6 +1,16 @@
 from django.conf import settings
 from django.contrib.auth.middleware import RemoteUserMiddleware
+from django.contrib.auth.backends import RemoteUserBackend
 
 
 class AuthMiddleware(RemoteUserMiddleware):
     header = settings.CURATION_PORTAL_AUTH_HEADER
+
+
+class AuthBackend(RemoteUserBackend):
+    def clean_username(self, username):
+        # See https://cloud.google.com/iap/docs/identity-howto
+        IAP_PREFIX = 'accounts.google.com:'
```

Comment:
Not a big deal for us, but could parameterise the prefix along with the header name.

Reply:
Yup, but kind of difficult to imagine what other prefixes we'd want to remove, so I'll wait until we need that flexibility :).